wagtail/utils/version.py

```diff
@@ -3,28 +3,45 @@
 
 
 def get_version(version):
-    "Returns a PEP 386-compliant version number from VERSION."
+    """Return a PEP 440-compliant version number from VERSION."""
+    version = get_complete_version(version)
+
     # Now build the two parts of the version number:
     # main = X.Y[.Z]
     # sub = .devN - for pre-alpha releases
-    #     | {a|b|c}N - for alpha, beta and rc releases
+    #     | {a|b|rc}N - for alpha, beta, and rc releases
 
     main = get_main_version(version)
 
     sub = ''
     if version[3] != 'final':
-        mapping = {'alpha': 'a', 'beta': 'b', 'rc': 'c'}
+        mapping = {'alpha': 'a', 'beta': 'b', 'rc': 'rc', 'dev': '.dev'}
         sub = mapping[version[3]] + str(version[4])
 
     return main + sub
 
 
-def get_main_version(version):
-    "Returns main version (X.Y[.Z]) from VERSION."
+def get_main_version(version=None):
+    """Return main version (X.Y[.Z]) from VERSION."""
+    version = get_complete_version(version)
     parts = 2 if version[2] == 0 else 3
     return '.'.join(str(x) for x in version[:parts])
 
 
+def get_complete_version(version=None):
+    """
+    Return a tuple of the Wagtail version. If version argument is non-empty,
+    check for correctness of the tuple provided.
+    """
+    if version is None:
+        from wagtail import VERSION as version
+    else:
+        assert len(version) == 5
```

Comment:
Is it ok to rely on asserts here? [They can be ignored](https://docs.python.org/3.6/using/cmdline.html#cmdoption-o).

Reply:
This bit was taken from Django. But I don't think this would matter as any changes to the version would be run through CI and that wouldn't ignore this statement.